missions/W5/M1/nyc_taxi_analysis.py

```
@@ -0,0 +1,48 @@
+from pyspark.sql import SparkSession
+from pyspark.sql import Row
+from pyspark.sql.functions import *
```

Ruff (0.8.2)

`from pyspark.sql.functions import *` used; unable to detect undefined names

(F403)